src/graphics/program-lib/chunks/normal.vert

```diff
@@ -6,6 +6,24 @@ vec3 getNormal() {
     #else
         dNormalMatrix = matrix_normal;
     #endif
-    return normalize(dNormalMatrix * vertex_normal);
+
+    vec3 tempNormal = vertex_normal;
+
+    #ifdef MORPHING
+        #ifdef MORPHING_NRM03
+            tempNormal += morpth_weights_a[0] * morph_nrm0;
```

Comment:
morpth_weights_a -> morph_weights_a

Reply:
fixing! hw did I missed that. At least it's consistent across all uses, so it works ;)